backend/Makefile

```diff
@@ -13,6 +18,17 @@ build:
 fmt:
 	go fmt ./...
 
+## migrations
+install_goose:
+	go get -u github.com/pressly/goose/v3/cmd/goose
+
+goose_up:
+	$(GOOSE_MIGRATION) up
```

Comment:
Where is the `GOOSE_MIGRATION` var set ?

Reply:
updated !